public/app/features/library-panels/components/PanelLibraryOptionsGroup/PanelLibraryOptionsGroup.tsx

```diff
@@ -23,7 +25,7 @@ export const PanelLibraryOptionsGroup: React.FC<Props> = ({ panel, dashboard })
     panel.restoreModel({
       ...panelInfo.model,
       ...pick(panel, 'gridPos', 'id'),
-      libraryPanel: pick(panelInfo, 'uid', 'name', 'meta'),
+      libraryPanel: toPanelModelLibraryPanel(panelInfo),
```

Comment:
I missed this in #31590 so any new library panels aren't updated correctly without this fix.